# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/PHttpChannel.ipdl
# Commit: 4214217c588c
# Full Hash: 4214217c588cd8f5a21acf197698905528f1ea17
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-30 10:00:42
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P5 allow OnProgress/OnStatus going thru pBg IPC, r=mayhemer,necko-reviewers
#   
#   Depends on D76971
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D76972
# ==============================================================================

diff -r e6c12eec1cef -r 4214217c588c netwerk/protocol/http/PHttpChannel.ipdl
--- a/netwerk/protocol/http/PHttpChannel.ipdl	Tue Jun 30 00:33:01 2020 +0000
+++ b/netwerk/protocol/http/PHttpChannel.ipdl	Tue Jun 30 00:33:19 2020 +0000
@@ -153,10 +153,6 @@
 
   async AltDataCacheInputStreamAvailable(IPCStream? stream);
 
-  async OnProgress(int64_t progress, int64_t progressMax);
-
-  async OnStatus(nsresult status);
-
   // Tell the child that the resource being loaded has been classified.
   async NotifyClassificationFlags(uint32_t aClassificationFlags, bool aIsThirdParty);
 
